boards/arm/nucleo_g070rb/nucleo_g070rb.yaml

```diff
@@ -0,0 +1,26 @@
+identifier: nucleo_g070rb
+name: ST Nucleo G070RB
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 36
+flash: 128
+supported:
+  - arduino_gpio
+  - arduino_i2c
+  - arduino_spi
+  - arduino_serial
+  - uart
+  - gpio
+  - i2c
+  - spi
+  - counter
+  - watchdog
+  - pwm
+  - adc
+  - dac
+  - dma
```

Comment:
DAC and DMA are not present in dts

Reply:
I removed DAC.

Regarding DMA, `nucleo_g071rb` which I based this PR on, also does not have dma in dts file.
It's actually defined in root file `stm32g051.dtsi`, which `nucleo_g071rb.dts` indirectly includes.

The same holds for `nucleo_g070rb` which indirectly includes `stm32g050.dtsi`.

Knowing this, is dma node required in yaml file?

Reply:
Adding dma will trigger dma test. So will require binding in dma tests. Not sure it is required, as this is already covered by `nucleo_g071rb`. We don't need each and single board to be tested/compiled